react/src/index.js

```diff
@@ -128,12 +128,20 @@ Sentry.init({
     });
 
     if (se) {
+      const se_tda_prefix_regex = /[^-]+-tda-[^-]+-/
+      let se_fp = se;
```

Comment:
nit: javascript casing might be more like `seTDAPrefixRegex`

also `se_fp` -- is `fp` fingerprint? If so, might as well `seFingerprint`?

Reply:
sure